src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -245,39 +244,37 @@ def verify_cont_test_result(self, result, expect):
         """
         if expect.lower() == 'pass':
             if DENY_ACCESS in result:
-                self.fail(
-                    "##Test Fail on verify_cont_test_result, expected Pass, "
-                    "but Failed.")
+                self.fail("##Test Fail on verify_cont_test_result, expected Pass, but Failed.")
             else:
-                self.log.info(
-                    " =Test Passed on verify_cont_test_result Succeed.\n")
+                self.log.info(" =Test Passed on verify_cont_test_result Succeed.\n")
         elif DENY_ACCESS not in result:
             self.fail(
-                "##Test Fail on verify_cont_test_result, expected Failure of "
-                "-1001, but Passed.")
+                "##Test Fail on verify_cont_test_result, expected Failure of -1001, but Passed.")
         else:
             self.log.info(
-                " =Test Passed on verify_cont_test_result expected error of "
-                "denial error -1001.\n")
+                " =Test Passed on verify_cont_test_result expected error of denial error -1001.\n")
 
-    def verify_cont_delete(self, expect):
+    def verify_cont_delete(self, container, expect):
         """verify container delete.
 
         Args:
+            container (TestContainer): container to verify.
             expect (str): expecting pass or deny.
         """
         action = "cont_delete"
-        result = self.destroy_test_container(self.pool.identifier, self.container.identifier)
+        daos = self.get_daos_command()
+        daos.exit_status_exception = False
+        result = daos.container_destroy(container.pool.identifier, container.identifier, True)
```

Comment:
Shouldn't `container.no_exception()` be used here?
```suggestion
        result = {}
        with container.no_exception():
            if container.destroy(force=1):
                result = container.daos.result
```

Reply:
Huh. I overlooked this one :) Will fix, thanks

Reply:
Except I think this
```
        with container.no_exception():
	            if container.destroy(force=1):
	                result = container.daos.result
```
Should just be
```
        with container.no_exception():
	            result = container.daos.result
```
to keep previous behavior of letting the test parse the result